bundles/framework/layerlist/Tile.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * @class Oskari.mapframework.bundle.layerlist.Tile
+ *
+ * Renders the "selected layers" tile.
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.layerlist.Tile',
```

Comment:
This should extend https://github.com/oskariorg/oskari-frontend/blob/master/bundles/framework/divmanazer/extension/DefaultTile.js so we can remove most of the methods in this class.